src/Pages/hyunchanpark/Main/FeedList.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import Feed from './Feed';
+
+class FeedList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      feedList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/feedData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          feedList: data,
+        });
+      });
+  }
+
+  onSubmitCommentForm = (e, feedId, newComment) => {
+    e.preventDefault();
+
+    if (newComment === '') return;
```

Comment:
빈 스트링은 falsy값이기때문에 아래와 같이 표현 가능합니다!
truthy falsy값 다시한번 확인해보시고 잘 활용하시면 좋겠습니다.
```suggestion
    if (!newComment) return;
```

Reply:
리뷰 반영했습니다.